utide/_reconstruct.py

```diff
@@ -54,7 +54,7 @@ def reconstruct(
         'min_SNR', and 'min_PE'.
         The input time array is included as 't_in', and 't_mpl';
         the former is the original input time argument, and the
-        latter is the time as a matplotlib datenum.  If 'epoch'
+        latter is the time as a datenum from '0000-12-31'.  If 'epoch'
```

Comment:
Including `t_mpl` was, in retrospect, a mistake on my part; I always hated that epoch, but never thought mpl would switch to a different one.
Suggestions: leave it in as-is but with a big warning in the docs that it is deprecated; and put in something like `t_dt64` that would be be a "datetime64[ms]" array, which will work perfectly for plotting with mpl.

Reply:
Agreed: t_mpl can be left in but deprecated; maybe removed in a year.  Ideally, it would generate a deprecation warning when accessed.  This would require turning it into a property.